packages/block-library/src/index.native.js

```diff
@@ -15,6 +15,9 @@ import * as more from './more';
 import * as paragraph from './paragraph';
 import * as image from './image';
 import * as nextpage from './nextpage';
+import * as UnsupportedBlock from './mobile/unsupported-block';
```

Comment:
I'm wondering whether it should be moved to `gutenberg-mobile` project. It's something that is specific only to the mobile project so doesn't quite makes sense here as it doesn't have web alternative.

Reply:
https://github.com/WordPress/gutenberg/pull/14577 's goal is to actually rename `unsupported-block` to `missing`, so we are in sync with the web part. I agree it might look confusing to have a rnmobile only block in the core block library but it isn't exposed to the web version and I think it's only temporary. We need those dependencies in `gutenberg` as they are required by other part of the code, see https://github.com/WordPress/gutenberg/pull/14662/files#diff-61ba183209a287f5b2da76defa6e20fbR7 and https://github.com/WordPress/gutenberg/pull/14662/files#diff-8aa4726165b0e0cdb260abe61e3329ceR10 for instance

Reply:
> #14577 's goal is to actually rename `unsupported-block` to `missing`, so we are in sync with the web part.

This would be awesome and exactly addresses my point.